scripts/dts/gen_defines.py

```diff
@@ -359,6 +361,24 @@ def write_compatibles(node):
             f"{node.z_path_id}_COMPAT_MATCHES_{str2ident(compat)}", 1)
 
 
+def write_child_identifiers(node):
+    # Writes macro that lists identifiers of all children per node.
+    if node.children:
+        labels = ''
+        # For each subnode with non-empty list of labels add the
+        # first (non-alias) label to the string of ', ' separated
+        # list of labels.
+        for subnode in node.children.values():
+            if subnode.labels:
+                labels = labels + subnode.labels[0]  + ', '
+
+        # Out list of labels only if non empty and remove trailing
+        # ', ' prior.
+        if labels:
+            out_dt_define(node.z_path_id + '_CHILD_IDENTIFIERS',
```

Comment:
shell we have a comment generated like for the parrent macro:

```suggestion
            out_comment(f"Node children ({node.path}) identifiers:")
            out_dt_define(node.z_path_id + '_CHILD_IDENTIFIERS',
```

Reply:
output checked, looks promising
good even without below improvement

Reply:
Done